MAINTAINERS.md

```diff
@@ -0,0 +1,79 @@
+# Maintainer Guidelines
+
+**This guide is for maintainers.** These people have **write
+access** to NREL HPC's repository and help merge the contributions of
+others.
+
+If you have something to contribute, please see the [Contributing Guide](CONTRIBUTING.md).
+
+This is a living document - if you see something out of date or missing,
+speak up!
+
+## What are a maintainer's responsibilities?
+
+It is every maintainer's responsibility to:
+
+* Expose a clear roadmap for improving the repository.
```

Comment:
"Expose" is a little vague. I think Lex may be the only project I've seen at NREL with a clear roadmap, and even that might be debated by those who were doing the work. Not sure what this bullet means.